specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/graphquery.json

```diff
@@ -267,6 +267,16 @@
           "readOnly": true,
           "description": "Azure resource name. This is GUID value. The display name should be assigned within properties field."
         },
+        "subscriptionId": {
```

Comment:
I do not understand the change here. This "Resource" definition was more or less aligned with  standard "TrackedResource" definition. But now you are adding subscription ID and resource group here, it is messing up this base definition. I would suggest this to go on to the specific query definition.

Also in terms of the name `GraphQueryResource`, if feels more like `XXXQuery` rather a `XXXResource`. Anyway, I guess that boat has sailed.

Reply:
Actually you are correct.. We are returning extra fields in the RP. Let me revert this change and fix our RP side.

Reply:
Reverted changes for subscriptionId/resourceGroup. Also fixed a casing issue for `etag` field.